src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -114,13 +122,15 @@ export const applicantContactInfoSchema = {
       'We’ll contact this phone number if we need to follow up about this form.',
     ),
     applicantPhone: phoneUI(),
+    contactEmail: emailUI(),
   },
   schema: {
     type: 'object',
     required: ['applicantPhone'],
     properties: {
       titleSchema,
       applicantPhone: phoneSchema,
+      contactEmail: emailSchema,
```

Comment:
Another small thing - can we have this field be `applicantEmail` and change [this one](https://github.com/department-of-veterans-affairs/vets-website/blob/dab5e87c1a1fc14898d689df1a930e6fb0b4cb82/src/applications/ivc-champva/10-7959C/chapters/formSignature.js#L45) from the previous PR to be `signerEmail`? That way the key names are consistent just to avoid confusion down the road.

Sorry for the back and forth on such a small change!